core/dbt/events/types.py

```diff
@@ -476,6 +476,16 @@ def message(self) -> str:
         return line_wrap_message(warning_tag(description))
 
 
+class MFCumulativeTypeParamsDeprecation(WarnLevel):
+    def code(self) -> str:
+        return "D019"
+
+    def message(self) -> str:
```

Comment:
Would it be easy to add the names of the outdated metrics in this message? Otherwise it might be hard for users to find.
But if that doesn't work with the behavior change objects easily, it probably isn't worth it.

Reply:
Ya, I don't think that is a straightforward change, given that it would change the interface. I think that could be an enhancement in the future.